src/pages/examples/form/ResponseContent.jsx

```diff
@@ -13,4 +14,13 @@ const ResponseContent = ({ response }) => (
   </div>
 );
 
+ResponseContent.propTypes = {
```

Comment:
https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/require-default-props.md
propType " response" is not required, but has no corresponding defaultProps declaration.

ResponseContent.propTypes = {
  response: PropTypes.shape({
    json: PropTypes.shape({
      name: PropTypes.string,
      password: PropTypes.string,
    }),
  }),
};

ResponseContent.defaultProps = {
  response: {
     json: {
       name: '',
       password: ''
     }
  }
}